gusto/linear_solvers.py

```diff
@@ -136,14 +139,18 @@ def _setup_solver(self):
         def V(u):
             return k*inner(u,k)
 
+        # specify degree for some terms as estimated degree is too large
+        dxp = dx(degree=(5,5))
```

Comment:
Extruded mesh so vertical and horizontal degree. 

I think Colin means, check the degree of the spaces and pick an appropriate degree that way: this under integrates a Q3 mass matrix, for example